rclpy/test/test_subscription.py

```diff
@@ -48,7 +49,13 @@ def make_mock_subscription(namespace, topic_name, cli_args=None):
     ('/example/topic', 'ns', '/example/topic'),
 ])
 def test_get_subscription_topic_name(topic_name, namespace, expected):
-    sub = make_mock_subscription(namespace, topic_name)
```

Comment:
make_mock_subscription() definition can be deleted from the file

Reply:
Opened https://github.com/ros2/rclpy/pull/809 deleting it from this file.